tests/test_publish.py

```diff
@@ -68,7 +70,7 @@ def fixture_publish_db():
     db['files'] = audformat.Table(db.files)
     db['files']['speaker'] = audformat.Column(scheme_id='speaker')
     db['files']['speaker'].set(
-        ['adam', 'adam', '11', '11'],
+        ['adam', 'adam', 'adam', '11'],
```

Comment:
Where is this change used in the updated tests?

Reply:
In the new tests I remove one file from the archive, alter one and leave one unchanged, see:

```
    alter_files = [
        'audio/001.wav',  # same archive as 'audio/00[2,3].wav'
        'audio/005.wav',
    ]
    rem_files = [
        'new/003.wav',  # same archive as 'audio/00[1,2].wav'
    ]
```

Reply:
But why do you have to change the speaker to `adam` then?

Reply:
To have all three files in one archive, before there were only two files in the archive called `adam`.